sdk/data/azappconfig/CHANGELOG.md

```diff
@@ -1,12 +1,15 @@
 # Release History
 
-## 0.4.4 (Unreleased)
+## 0.5.0 (Unreleased)
```

Comment:
Isn't there a constant that needs to be updated too?

Reply:
You are right, and the CI just failed for that reason. I just updated it in the most recent commit.